cleanrl/ppo_dna_atari_envpool.py

```diff
@@ -0,0 +1,468 @@
+# DNA: Proximal Policy Optimization with a Dual Network Architecture
+# https://arxiv.org/abs/2206.10027
+
+import argparse
+import os
+import random
+import time
+from collections import deque
+from distutils.util import strtobool
+
+import envpool
+import gym
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.distributions import Categorical, kl_divergence
+from torch.utils.tensorboard import SummaryWriter
+
+
+def parse_args():
+    # fmt: off
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--exp-name", type=str, default=os.path.basename(__file__).rstrip(".py"),
+        help="the name of this experiment")
+    parser.add_argument("--seed", type=int, default=np.random.randint(2 ** 31),
+        help="seed of the experiment")
+    parser.add_argument("--torch-deterministic", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="if toggled, `torch.backends.cudnn.deterministic=False`")
+    parser.add_argument("--cuda", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="if toggled, cuda will be enabled by default")
+    parser.add_argument("--track", type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="if toggled, this experiment will be tracked with Weights and Biases")
+    parser.add_argument("--wandb-project-name", type=str, default="cleanRL",
+        help="the wandb's project name")
+    parser.add_argument("--wandb-entity", type=str, default=None,
+        help="the entity (team) of wandb's project")
+
+    # Algorithm specific arguments
+    parser.add_argument("--env-id", type=str, default="Pong-v5",
+        help="the id of the environment")
+    parser.add_argument("--total-timesteps", type=int, default=10_000_000,
```

Comment:
Here we use the style `10000000`. I'd be open to using `10_000_000`, but this change should apply to all of our variants in the form of another PR.

 It's probably best to automate this using something like a pre-commit hook that automatically converts all instances of `10000000` to `10_000_000`.

Reply:
👍